src/blocks/common.blocks/search-form/search-form.browser.js

```diff
@@ -0,0 +1,19 @@
+modules.define('search-form', ['i-bem__dom'], function (provide, BEMDOM) {
+    provide(BEMDOM.decl(this.name, {
+        onSetMod: {
+            js: {
+                inited: function () {
+                    this._input = this.findBlockInside('input', 'input');
+                }
+            }
+        },
+
+        activate: function () {
+            var _this = this;
+
+            setTimeout(function () {
```

Comment:
It is necessary to put the focus after opening a panel and fix a jolt http://g.recordit.co/bkZycCBYJn.gif